array-methods/for-each.js

```diff
@@ -0,0 +1,12 @@
+const values = [10, 1, 22, 23, 41, 5, 18, 7, 80, 9];
+
+console.log('In order: ');
+
+values.forEach((element) => console.log(element));
+
+console.log('Reverse Order: ');
+
+values.forEach((_, index, element) => {
+  const reverseIndex = element.length - 1 - index;
+  console.log(element[reverseIndex]);
+});
```

Comment:
`element` should be your 1st argument, and you should not have a 3rd argument. You want `element` to represent the value inside the array, and not the array itself